packages/react-app/src/components/Header.tsx

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 import { ConnectWalletButton } from ".";
+import { useAppSelector } from "../hooks";
+import { RootState } from "../store";
 
 interface Props {
   connectWallet: () => void;
+  userNft: number;
 }
-export default function Header({ connectWallet }: Props) {
+export default function Header({ connectWallet, userNft }: Props) {
+  const userAddress = useAppSelector((state: RootState) => state.user.address);
   return (
     <div className="connect-wallet-section header border-b primary-font text-xl flex h-16 lg:px-4 items-center">
-      <span className="bg-gray-1 opacity-75 text-lg secondary-font mr-5">YOUR PLOTS (0)</span>
+      {userAddress && (
+        <span className="bg-gray-1 opacity-75 text-lg secondary-font mr-5">YOUR PLOTS ({userNft})</span>
```

Comment:
userNft displays 0 even I own parcel-0 NFT until the readOwnedParcelCount call finished in the parent component.